packages/apps/shopify-app-remix/docs/staticPages/webhooks.doc.ts

```diff
@@ -9,15 +9,37 @@ const data: LandingTemplateSchema = {
     {
       type: 'Generic',
       anchorLink: 'config',
-      title: 'Configuring webhooks subscriptions',
+      title: 'Subscribe using the app configuration file',
+      sectionContent:
+        'The easiest and recommended way to configure your webhooks is to edit your app configuration file. ' +
+        'You can find more info in the [webhooks documentation](/docs/apps/webhooks/getting-started-declarative).' +
+        '\n\nTo set up a simple HTTPS webhook subscription, you can follow these steps:' +
+        '\n1. Add in `shopify.app.toml` the topic you want to subscribe to. In this example we subscribe to the `APP_UNINSTALLED` topic.' +
+        '\n1. Review the required scopes for the webhook topics, and update your [app scopes](/docs/apps/tools/cli/configuration#access_scopes) as necessary.' +
+        '\n1. Run `shopify app deploy` from the CLI to save your webhooks configuration.',
+      codeblock: {
+        title: 'Configure webhooks subscriptions via app configuration',
+        tabs: [
+          {
+            title: 'shopify.app.toml',
+            code: './examples/guides/webhooks/declarative-config.example.ts',
+            language: 'tsx',
+          },
+        ],
+      },
+    },
+    {
+      type: 'Generic',
+      anchorLink: 'config',
+      title: 'Subscribe using the API',
       sectionContent:
         'Configure `shopifyApp` and setup webhook subscription with the following steps:' +
         '\n1. The webhooks you want to subscribe to. In this example we subscribe to the `APP_UNINSTALLED` topic.' +
         '\n1. The code to register the `APP_UNINSTALLED` topic after a merchant installs you app. Here `shopifyApp` provides an `afterAuth` hook.' +
-        '\n1. Review the required scopes for the webhook topics, and update your [app scopes](/docs/apps/tools/cli/configuration#access_scopes)  as necessary.' +
+        '\n1. Review the required scopes for the webhook topics, and update your [app scopes](/docs/apps/tools/cli/configuration#access_scopes) as necessary.' +
         "\n\n> Note: You can't register mandatory topics using this package, you must [configure those in the Partner Dashboard](/docs/apps/webhooks/configuration/mandatory-webhooks) instead.",
       codeblock: {
-        title: 'Configure webhooks subscriptions',
+        title: 'Configure webhooks subscriptions via API',
```

Comment:
Nit: I know on Shopify.dev we try not to use `via`. (Only know this because I have tried many times to do so. )
```suggestion
        title: 'Configure webhooks subscriptions with the API',
```

Reply:
Updated, thanks!